build/ci/send-to-helix.yml

```diff
@@ -20,6 +20,31 @@ parameters:
   condition: succeeded()                 # optional -- condition for step to execute; defaults to succeeded()
 
 steps:
+  - script: ${{ parameters.MsBuildScript}}
+            $(Build.SourcesDirectory)/eng/helix.proj
+            /t:Restore
+            /bl:$(Build.SourcesDirectory)/artifacts/log/${{ parameters.Configuration }}/SendToHelix.binlog
```

Comment:
This uses the same log as below, also most of these parameters are not needed for restore.  I imagine this step is just temporary until we refactor in a follow up PR though.